lib/ts/recipe/session/api/implementation.ts

```diff
@@ -13,7 +14,20 @@ export default function getAPIInterface(): APIInterface {
         }: {
             options: APIOptions;
             userContext: UserContext;
-        }): Promise<SessionContainerInterface> {
+        }): Promise<SessionContainerInterface | undefined> {
```

Comment:
I think refresh should throw if we detect duplicate tokens but `olderCookieDomain` is undefined

Reply:
Yeah, I agreed.  Updated the PR to address this.